src/swap/v2/index.ts

```diff
@@ -347,6 +406,35 @@ function getFixedOutputSwapQuote({
       convertFromBaseUnits(decimals.assetIn, Number(swapInputAmount)),
     priceImpact
   };
+
+  if (isSwapRouterEnabled) {
+    const swapRoute = await getSwapRoute({
+      amount: assetOut.amount,
+      assetInID,
+      assetOutID: assetOut.id,
+      swapType: SwapType.FixedInput,
+      network
+    });
+
+    if (
+      swapRoute.route.length > 1 &&
+      BigInt(swapRoute.route[swapRoute.route.length - 1].quote.amount_in.amount) <
```

Comment:
```suggestion
      BigInt(swapRoute.route[swapRoute.route.length - 1].quote.amount_in.amount) <
```
It's hard to understand what this value is, maybe we can create a constant for this so it'll be more clear;